bouncy/release-bionic-arm64-build.yaml

```diff
@@ -0,0 +1,91 @@
+%YAML 1.1
+# ROS buildfarm release-build file
+---
+abi_incompatibility_assumed: true
+jenkins_binary_job_label: buildagent_arm64 || bouncy_binarydeb_ubv8
+jenkins_binary_job_priority: 82
+jenkins_binary_job_timeout: 720
+jenkins_source_job_priority: 78
+jenkins_source_job_timeout: 30
+notifications:
+  emails:
+  - mikael+build.ros2.org@openrobotics.org
+  - ros2-buildfarm-bouncy@googlegroups.com
+  maintainers: false
```

Comment:
Is it intentional that these are disabled? Or when are they planned to be enabled if not now?

Reply:
> Is it intentional that these are disabled?

Yes

>  Or when are they planned to be enabled if not now?

When we have email enabled on the ROS2 farm.

From https://github.com/ros2/ros_buildfarm_config/pull/5#issuecomment-393350851: `Once we have the other things sorted out like the smtp set up`

The task of configuring emails is tracked on the [release checklist](https://github.com/ros2/ros2/issues/493) under: "Infrastructure/configure build.ros2.org smtp"

Reply:
I don't see any harm in enabling this option now. The configuration already lists other emails for notification so Jenkins is going to try sending an email anyway.

Reply:
Currently it is disabled in all build files of build.ros2.org.

As we didn't have email configured, I didn't see value in enabling it only in this file and open a follow-up to modify other (ardent) of incoming (source) files, neither it felt like this PR should touch unrelated files like the ardent ones.

If you think now is a good time, let's open a PR to change all files that should have this enabled :+1:

Reply:
> As we didn't have email configured, I didn't see value in enabling it only in this file

Well, I just saw it from the opposite side: there is no reason not to enable it - Jenkins is trying to send notifications anyway. So it just needs to enabled later. I will leave it up to you to enable it whenever you like.

Reply:
The email notification still needs to be enabled. The maintainers are currently not being notified about the outcome of the release builds.